sdk/python/kfp/_client.py

```diff
@@ -108,7 +108,7 @@ def create_experiment(self, name):
     """
     import kfp_experiment
 
-    experiment = self.get_experiment(experiment_name=name)
+    experiment = self.get_experiment(experiment_name=name, ignore_error=True)
```

Comment:
AFAIK, the pythonic way to accomplish this is to just use `try`/`except`.

Reply:
You are right. Done.